include/libtorrent/alert_types.hpp

```diff
@@ -1338,6 +1338,7 @@ namespace libtorrent
 	// Port 0 asks the operating system to pick a port that's free). If that fails
 	// you may see a listen_failed_alert with port 0 even if you didn't ask to
 	// listen on it.
+#undef interface
```

Comment:
could you leave a comment here explaining why this needs to be undefined? I take it some winrt header defines it. Is there no way to make it shut-up? similar to WIN32_MEAN_AND_LEAN.

Also, would it work if this moved into libtorrent/config.hpp?